script/Arcueid/Day 9/09_00_ARC09_3_1.txt

```diff
@@ -162,12 +175,14 @@ The exterior of the gazebo built in the courtyard.
 [sha:3dd84df21bfb598938f72c8d8e1c867f10afaf2f]{
 -- Page 110, Offset 21580.
 -- 柱の下、陽射しの影に隠れて見つからない場所に、石を削って書かれた文字がある。
-In a place concealed by the shadows cast by the sunlight, there are words carved in stone on the bottom of a pillar.
+Low down on the columns,
+ in the nooks and crannies where the sun's rays can't quite reach,
+ you can still see where letters have been scraped into the stone.
 }
 [sha:9402b59e34eafad828fb1417c780dd077fb0aa29]{
 -- Page 111, Offset 21581.
 -- ガゼボをぐるりと回って、もうかすれている文字をカウントする。シキ、アキハ、シキ、シキ、シキ。
-I circle around the gazebo, counting the words that have already been blurred. Shiki, Akiha, Shiki, Shiki, Shiki.
+I circle around the gazebo, tallying up the crumbling words. Shiki, Akiha, Shiki, Shiki, Shiki.
```

Comment:
Oops, yeah according to our previous discussion this should probably be _Italics_ since these are katakana, and the _Shiki_ are presumably a mix of Shiki and SHIKI